kiva/pdf.py

```diff
@@ -623,9 +623,10 @@ def set_font(self, font):
         """ Sets the font for the current graphics context.
         """
         # TODO: Make this actually do the right thing
```

Comment:
Is this TODO still relevant? Perhaps the right thing is not choosing Helvetica.

Reply:
I have no idea what the context was for this comment... But as I'm in Switzerland right now, I'm pretty sure I'll get kicked out if I use anything but Helvetica (Arial doesn't count!)